README.md

```diff
@@ -1,54 +1,104 @@
 3. [Taskfile](https://taskfile.dev/installation)
-4. Protocol buffers go plugins and other command line tools. The `tool` task will install the required versions of the tools into your `GOPATH`, so be sure to include `GOPATH/bin` on your `PATH`.
+4. [Docker](https://docs.docker.com/get-docker/) to run a PostgreSQL database.
+5. Miscellaneous go plugins and other command line tools. The `tool` task will install the required versions of the tools into your `GOPATH`, so be sure to include `GOPATH/bin` on your `PATH`.
 
     ```shell
     task tools
     ```
 
-### Build
+#### Build
 
-Invoke `task` command to see all available tasks. The `build` task will compile `kwild`, `kwil-cli`, and `kwil-admin`. They will be generated in `.build/`:
+Invoke `task` command to see all available tasks. The `build` task will compile `kwild`, `kwil-cli`, and `kwil-admin` binaries. They will be generated in `.build/`:
 
 ```shell
 task build
 ```
 
-## Local deployment
+### Local Deployment
+
+#### Start PostgreSQL
+
+Each Kwil node requires a PostgreSQL instance to run. You can start a PostgreSQL database using Docker Compose:
+
+```shell
+docker compose -f ./deployments/compose/postgres/docker-compose.yml up
+```
+
+### Start kwild
 
 You can start a single node network using the `kwild` binary built in the step above:
 
 ```shell
-.build/kwild --autogen
+.build/kwild --autogen --app.pg_db_host localhost
 ```
 
-For more information on running nodes, and how to run a multi-node network, refer to the Kwil [documentation](<https://docs.kwil.com/docs/node/quickstart>).
+For more information on running nodes, and how to run a multi-node network, refer to the Kwil [documentation](https://docs.kwil.com/docs/node/quickstart).
 
-## Resetting local deployments
+### Resetting local deployments
 
 By default, `kwild` stores all data in `~/.kwild`. To reset the data on a deployment, remove the data directory while the node is stopped:
 
 ```shell
 rm -r ~/.kwild
 ```
 
+## Extensions
+
+Kwil offers an extension system that allows you to extend the functionality of your network (e.g. building network oracles, customizing authentication, creating network deterministic compute, etc.). To learn more about the types of extensions and how to build them, refer to the extensions directory [README](extensions/README.md).
+
+## Contributing
+
+We welcome contributions to kwil-db. To contribute, please read our [contributing guidelines](CONTRIBUTING.md).
+
 ## License
 
 The kwil-db repository (i.e. everything outside of the `core` directory) is licensed under the Apache License, Version 2.0. See [LICENSE](LICENSE) for more details.
 
-The kwil golang SDK (i.e. everything inside of the `core` directory) is licensed under the MIT License. See [core/LICENSE.md](core/LICENSE.md) for more details.
+The kwil Golang SDK (i.e. everything inside of the `core` directory) is licensed under the MIT License. See [core/LICENSE.md](core/LICENSE.md) for more details.
```

Comment:
"The Kwil Go SDK"
Sometimes "golang" is helpful to disambiguate, but: https://go.dev/doc/faq#go_or_golang
So here's it's properly just a "Go SDK"

Reply:
Ah i did not know that. Thanks!